CDP4Common/CDP4Common.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1</TargetFrameworks>
     <Company>RHEA System S.A.</Company>
     <Title>CDP4Common Community Edition</Title>
-    <VersionPrefix>8.3.0</VersionPrefix>
+    <VersionPrefix>8.3.1</VersionPrefix>

```

Comment:
because we change the inheritance of the exceptions this is going to be a major release, so please bump to 9.0.0